manticore/manticore.py

```diff
@@ -294,37 +268,6 @@ def maxstorage(self):
     def maxstorage(self, max_storage):
         self._maxstorage = max_storage
 
-    @property
-    def verbosity(self):
```

Comment:
Can we keep this API? `m.verbosity = x` feels more idiomatic to me than `m.set_verbosity(x)`